pkg/networkservice/chains/nsmgr/server_test.go

```diff
@@ -318,3 +328,17 @@ func (p *passThroughClient) Close(ctx context.Context, conn *networkservice.Conn
 	conn = conn.Clone()
 	return next.Client(ctx).Close(ctx, conn, opts...)
 }
+
+type counterServer struct {
+	Requests, Closes int
```

Comment:
Please use atomic int32 to prevent race condition in case of a testing parallel request from a few clients